src/ed_object_models/spawn_sdf_in_gazebo.py

```diff
@@ -5,18 +5,15 @@
 import os
 import glob
 
-from rospkg import RosPack
 from gazebo_msgs.srv import SpawnModel
-from geometry_msgs.msg import Pose
+from geometry_msgs.msg import *
```

Comment:
Please don't import everything in the current scope. Either use ```ìmport BlaatBlaatBlaat as bla``` or ```from BlaatBlaatBlaat import bladiebla```.
Please use second option here.

Reply:
I initially thought that PEP8 stated that multiple imports on the same line was bad, but just saw multiple module import from the same package is OK. So I have updated it accordingly.